updated_tictactoe.py

```diff
@@ -0,0 +1,146 @@
+import sys
+
+
+def format_user(element):
+    """
+    Return a string representation of a board position that can be logged to console.
+    """
+    if element is None:
+        return '_'
+    elif element:
+        return 'x'
+    else:
+        return 'o'
+
+
+def print_board(board):
+    """
+    Print the board to console.
+    """
+    for row in board:
+        for position in row:
+            print(f"{format_user(position)} ", end="")
+        print()
+
+
+def get_user_move():
+    """
+    Get the users move as an input from the console.  This method valids that the number
+    is between 1 - 9 or "q" (for quiting the game).
+    """
+    user_input = input("Please enter a position 1 through 9 or enter \"q\" to quit: ")
+
+    # Simply quit the program.
+    if user_input == 'q':
+        sys.exit()
```

Comment:
It says "sys" is not defined. Is this because I have not imported sys at the top?

Reply:
Yep. `sys` is a package in the standard Python library. This means you don't have to install it with pip but you do have to import it when you want to use it within your code.